### DISCUSSION TITLE
Error in advanced-01-open-telemetry: Cannot run the provider, consumer and Jaeger

### DISCUSSION THREAD

**Discussion by ghost** [eclipse-edc/Samples#246] [Q&A]
When I try to execute: docker compose -f advanced/advanced-01-open-telemetry/docker-compose.yaml up --abort-on-container-exit

I always get this: consumer-1    | 2024-04-20 16:59:26 FINE : Exchange request line: GET /metrics HTTP/1.1 
consumer-1    | 2024-04-20 16:59:26 FINE : GET /metrics HTTP/1.1 [200  OK] () 
provider-1    | 2024-04-20 16:59:26 FINE : [HttpClient-1-SelectorManager] [549s 877ms] HttpClientImpl(1) next timeout: 0 
provider-1    | 2024-04-20 16:59:26 FINE : [HttpClient-1-SelectorManager] [549s 878ms] HttpClientImpl(1) next expired: 0 
provider-1    | 2024-04-20 16:59:26 FINE : [HttpClient-1-SelectorManager] [549s 878ms] HttpClientImpl(1) Next deadline is 3000 
consumer-1    | 2024-04-20 16:59:26 FINE : [HttpClient-1-SelectorManager] [549s 881ms] HttpClientImpl(1) next timeout: 0 
consumer-1    | 2024-04-20 16:59:26 FINE : [HttpClient-1-SelectorManager] [549s 881ms] HttpClientImpl(1) next expired: 0 
consumer-1    | 2024-04-20 16:59:26 FINE : [HttpClient-1-SelectorManager] [549s 881ms] HttpClientImpl(1) Next deadline is 3000 
consumer-1    | 2024-04-20 16:59:29 FINE : [HttpClient-1-SelectorManager] [552s 884ms] HttpClientImpl(1) next timeout: 0 
consumer-1    | 2024-04-20 16:59:29 FINE : [HttpClient-1-SelectorManager] [552s 884ms] HttpClientImpl(1) next expired: 0 
consumer-1    | 2024-04-20 16:59:29 FINE : [HttpClient-1-SelectorManager] [552s 884ms] HttpClientImpl(1) Next deadline is 3000 
provider-1    | 2024-04-20 16:59:29 FINE : [HttpClient-1-SelectorManager] [552s 881ms] HttpClientImpl(1) next timeout: 0 
provider-1    | 2024-04-20 16:59:29 FINE : [HttpClient-1-SelectorManager] [552s 882ms] HttpClientImpl(1) next expired: 0 
provider-1    | 2024-04-20 16:59:29 FINE : [HttpClient-1-SelectorManager] [552s 882ms] HttpClientImpl(1) Next deadline is 3000 
consumer-1    | 2024-04-20 16:59:32 FINE : [HttpClient-1-SelectorManager] [555s 885ms] HttpClientImpl(1) next timeout: 0 
consumer-1    | 2024-04-20 16:59:32 FINE : [HttpClient-1-SelectorManager] [555s 885ms] HttpClientImpl(1) next expired: 0 
consumer-1    | 2024-04-20 16:59:32 FINE : [HttpClient-1-SelectorManager] [555s 885ms] HttpClientImpl(1) Next deadline is 3000 
provider-1    | 2024-04-20 16:59:32 FINE : [HttpClient-1-SelectorManager] [555s 884ms] HttpClientImpl(1) next timeout: 0 
provider-1    | 2024-04-20 16:59:32 FINE : [HttpClient-1-SelectorManager] [555s 884ms] HttpClientImpl(1) next expired: 0 
provider-1    | 2024-04-20 16:59:32 FINE : [HttpClient-1-SelectorManager] [555s 884ms] HttpClientImpl(1) Next deadline is 3000 
consumer-1    | 2024-04-20 16:59:35 FINE : [HttpClient-1-SelectorManager] [558s 888ms] HttpClientImpl(1) next timeout: 0 
consumer-1    | 2024-04-20 16:59:35 FINE : [HttpClient-1-SelectorManager] [558s 889ms] HttpClientImpl(1) next expired: 0 
consumer-1    | 2024-04-20 16:59:35 FINE : [HttpClient-1-SelectorManager] [558s 889ms] HttpClientImpl(1) Next deadline is 3000 
provider-1    | 2024-04-20 16:59:35 FINE : [HttpClient-1-SelectorManager] [558s 887ms] HttpClientImpl(1) next timeout: 0 
provider-1    | 2024-04-20 16:59:35 FINE : [HttpClient-1-SelectorManager] [558s 888ms] HttpClientImpl(1) next expired: 0 
provider-1    | 2024-04-20 16:59:35 FINE : [HttpClient-1-SelectorManager] [558s 888ms] HttpClientImpl(1) Next deadline is 3000 
consumer-1    | 2024-04-20 16:59:36 FINE : Exchange request line: GET /metrics HTTP/1.1 
consumer-1    | 2024-04-20 16:59:36 FINE : GET /metrics HTTP/1.1 [200  OK] () 
consumer-1    | 2024-04-20 16:59:38 FINE : [HttpClient-1-SelectorManager] [561s 892ms] HttpClientImpl(1) next timeout: 0 
consumer-1    | 2024-04-20 16:59:38 FINE : [HttpClient-1-SelectorManager] [561s 892ms] HttpClientImpl(1) next expired: 0 
consumer-1    | 2024-04-20 16:59:38 FINE : [HttpClient-1-SelectorManager] [561s 892ms] HttpClientImpl(1) Next deadline is 3000 
provider-1    | 2024-04-20 16:59:38 FINE : [HttpClient-1-SelectorManager] [561s 897ms] HttpClientImpl(1) next timeout: 0 
provider-1    | 2024-04-20 16:59:38 FINE : [HttpClient-1-SelectorManager] [561s 897ms] HttpClientImpl(1) next expired: 0 
provider-1    | 2024-04-20 16:59:38 FINE : [HttpClient-1-SelectorManager] [561s 897ms] HttpClientImpl(1) Next deadline is 3000 
consumer-1    | 2024-04-20 16:59:41 FINE : [HttpClient-1-SelectorManager] [564s 895ms] HttpClientImpl(1) next timeout: 0 
consumer-1    | 2024-04-20 16:59:41 FINE : [HttpClient-1-SelectorManager] [564s 895ms] HttpClientImpl(1) next expired: 0 
consumer-1    | 2024-04-20 16:59:41 FINE : [HttpClient-1-SelectorManager] [564s 895ms] HttpClientImpl(1) Next deadline is 3000 
provider-1    | 2024-04-20 16:59:41 FINE : [HttpClient-1-SelectorManager] [564s 898ms] HttpClientImpl(1) next timeout: 0 
provider-1    | 2024-04-20 16:59:41 FINE : [HttpClient-1-SelectorManager] [564s 898ms] HttpClientImpl(1) next expired: 0 
provider-1    | 2024-04-20 16:59:41 FINE : [HttpClient-1-SelectorManager] [564s 898ms] HttpClientImpl(1) Next deadline is 3000 
consumer-1    | 2024-04-20 16:59:44 FINE : [HttpClient-1-SelectorManager] [567s 897ms] HttpClientImpl(1) next timeout: 0 
consumer-1    | 2024-04-20 16:59:44 FINE : [HttpClient-1-SelectorManager] [567s 897ms] HttpClientImpl(1) next expired: 0 
consumer-1    | 2024-04-20 16:59:44 FINE : [HttpClient-1-SelectorManager] [567s 897ms] HttpClientImpl(1) Next deadline is 3000 
provider-1    | 2024-04-20 16:59:44 FINE : [HttpClient-1-SelectorManager] [567s 901ms] HttpClientImpl(1) next timeout: 0 
provider-1    | 2024-04-20 16:59:44 FINE : [HttpClient-1-SelectorManager] [567s 901ms] HttpClientImpl(1) next expired: 0 
provider-1    | 2024-04-20 16:59:44 FINE : [HttpClient-1-SelectorManager] [567s 901ms] HttpClientImpl(1) Next deadline is 3000 
 Can anybody help me? Thank you

**Comment by jimmarino** [comment]
Without a stacktrace or additional information, it is impossible to understand what the issue is.

**Reply by ghost** [reply]
I want to run the consumer, the provider, and the Jaeger with the command: docker compose -f advanced/advanced-01-open-telemetry/docker-compose.yaml up --abort-on-container-exit

However, the connection to port localhost 4318 is refused:
![grafik](https://github.com/eclipse-edc/Connector/assets/160601018/6b0c4ca8-97e1-41c1-b9ad-4812a4dbd5f0)
